core/src/main/resources/lib/layout/task.jelly

```diff
@@ -79,19 +79,20 @@ THE SOFTWARE.
     The semantics is that when one of the nested <task> matches, the parent is also considered as a match
     and thus gets a highlight.
   -->
-  <j:set var="match" value="${h.hyperlinkMatchesCurrentPage(href)}" />
+  <j:set var="isCurrent" value="${h.hyperlinkMatchesCurrentPage(href)}" />
   <j:scope>
-    <j:set var="taskMatching" value="${context.parent}"/>
+    <j:set var="parentTagContext" value="${context.parent}"/>
     <d:invokeBody />
   </j:scope>
 
-  <j:if test="${attrs.permission==null or h.hasPermission(it,attrs.permission)}">
+  <j:if test="${href != 'newDomain' and attrs.permission==null or h.hasPermission(it,attrs.permission)}">
```

Comment:
what's this newDomain check for?

Reply:
oops I think that's dev code that I forgot to remove, it should be fixed now